src/net_processing.cpp

```diff
@@ -3206,6 +3206,10 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
             return;
         }
 
+        // Also stop processing the transaction early if we are still in IBD since we don't
```

Comment:
Perhaps remove "Also". Comments have a habit of migrating around and not getting updated, so the "also" that this is referring to may no longer make sense in future.

Reply:
Done!